Gordon360/Models/CCT_DB_Models.edmx.diagram

```diff
@@ -4,41 +4,51 @@
   <edmx:Designer xmlns="http://schemas.microsoft.com/ado/2009/11/edmx">
```

Comment:
This is a necessary part of rebuilding the edmx files. That is an automated process as part of Entity Framework that builds auto-generated classes etc. based on the artifacts in our database. So when we update the database, we have to rebuild the edmx files to pull in the database changes. Since they are auto-generated, they shouldn't cause any issues.